app/views/content/blog/no-two-days-are-the-same.md

```diff
@@ -0,0 +1,43 @@
+---
+title: No two days are the same
+date: "2020-01-13"
+author: Gavin McIntyre
+images:
+  q-gavin:
+    path: "media/images/content/blog/stories-gavin.jpg"
+    thumbnail_path: "media/images/content/blog/stories-gavin.jpg"
+description: |-
+  Assistant headteacher Gavin McIntyre shares his journey to become a teacher.
+keywords:
+  - life as a teacher
+  - day in the life
+  - classroom
```

Comment:
need to change this tag - needs to be make sense after 'Blog posts about' as this is the title on the blog tag pages